gitpod.qmd

```diff
@@ -1,6 +1,19 @@
 # Gitpod
 
-## What is Gitpod (Estelle/ Cormac to review)
+## What is Gitpod
+
+Gitpod is a developer environment that runs in your browser. In order to use it, you need a Git repository (either on GitHub, GitLab or Bitbucket). Gitpod opens up a container on your Git repository.
```

Comment:
Could mention that it builds the container based on the ".gitpod.yml" file found in the repo?